layers/cmd/valueset.go

```diff
@@ -216,6 +220,26 @@ func (vsets Valuesets) Lookup(ctx *ls.Context, req ls.ValuesetLookupRequest) (ls
 			if err := lookup(v); err != nil {
 				return ls.ValuesetLookupResponse{}, err
 			}
+		} else if v, ok := vsets.Services[id]; ok {
```

Comment:
First go through the list of table ids and find out which ones are for service calls.
For each of those, call the corresponding service.
If multiple table ids map to the same service, call that service with multiple ids.

Example: 
Request:  t1, t2, t3, t4
t2 -> svc1
t3, t4 -> svc2
Lookup t1 in the valueset in mem
call svc1 with tableid=t1
call svc2 with tableid=t3,t4